OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
     <DebugSymbols>true</DebugSymbols>
-    <OutputPath>..\</OutputPath>
+    <OutputPath>bin\Debug\</OutputPath>
```

Comment:
I had to change it to bin/Debug and forces references to be copied into the bin/Debug folder due to conflicts with OpenRA.Mods.Common placed in root, and inside OpenRA.Mods.Common/bin/Debug. I would leave it as it is now.